migrations/anser.go

```diff
@@ -73,22 +81,48 @@ func (opts Options) Application(env anser.Environment, evgEnv evergreen.Environm
 		return nil, err
 	}
 
-	generatorFactories := []migrationGeneratorFactory{
-		//addExecutionToTasksGenerator,
-		//oldTestResultsGenerator,
-		//testResultsGenerator,
-		projectAliasesToCollectionGenerator,
-		githubHooksToCollectionGenerator,
-		zeroDateFixGenerator(githubToken),
-		adminEventRestructureGenerator,
+	generatorFactories := map[string]migrationGeneratorFactory{
+		// Early Migrations, disabled because the generator queries are not properly indexed.
+		//
+		// "migration-testresults-legacy-no-execution": addExecutionToTasksGenerator,
+		// "migration-testresults-oldtasks": oldTestResultsGenerator,
+		// "migration-testresults-tasks": testResultsGenerator,
+
+		"migration-project-aliases-to-collection": projectAliasesToCollectionGenerator,
+		"migration-github-hooks-to-collection":    githubHooksToCollectionGenerator,
```

Comment:
I think it would be better to define the migration names in the same files that implement them